packages/framework-core/src/services/pub-sub/read-model-pub-sub.ts

```diff
@@ -4,6 +4,7 @@ import {
   ReadModelInterface,
   ReadModelPropertyFilter,
   Instance,
+  Operation,
```

Comment:
@adriangmweb This file should also be adapted to the new filter structure, but I see it keeps the same old structure, so subscriptions won't work properly.

_Maybe you want to do this in another PR? If so, please create the issue_

Context: When you subscribe to a ReadModel, you use the same filter as when querying them. We need to implement the behavior of those filters (as you did in AWS) but, this time, by filtering the array of read models in memory, in pure javascript (instead of with AWS SQL).

The GraphQL engine will call this code with an array of ReadModels of a specific type that are eligible to send to the user who subscribed, but we need to filter them using the filters the user specified in the Subscriptoin opreation.

Reply:
This won't actually be able to test until we got the GraphQL new filter working, right? I'll create the issue to solve it along with the new schema

Reply:
Here's the issue https://github.com/boostercloud/booster/issues/602